qir/qat/Apps/CMakeLists.txt

```diff
@@ -1,4 +1,10 @@
 add_executable(qat Qat/Qat.cpp Qat/QatConfig.cpp)
 
 target_link_libraries(qat ${llvm_libs})
-target_link_libraries(qat Logging TransformationRulesPass Rules AllocationManager Commandline Generators Profile Validator ValidationPass StaticResourceComponent GroupingPass ModuleLoader PostTransformValidation PreTransformTrimming ZExtTransformPass RecordPullBackPass)
+target_link_libraries(qat Logging TransformationRulesPass Rules AllocationManager Commandline Generators Profile Validator ValidationPass StaticResourceComponent GroupingPass ModuleLoader PostTransformValidation PreTransformTrimming ZExtTransformPass RecordPullBackPass Version)
```

Comment:
* The long fragment `Logging TransformationRulesPass . . . RecordPullBackPass Version` occurs more than once. Consider de-duplicating.

Reply:
The long list should not have been included in the second target.